marlowe-runtime/src/Language/Marlowe/Runtime/Transaction/Server.hs

```diff
@@ -25,13 +25,14 @@ import Cardano.Api
   , makeShelleyAddress
```

Comment:
This file contains substantive changes.

Reply:
Nice example of a Marlowe sync client - I've just learned a lot about the API from this example.